dali/operators/image/paste/multipaste.cc

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/image/paste/multipaste.h"
+#include "dali/kernels/imgproc/paste/paste.h"
+#include "dali/core/tensor_view.h"
+
+namespace dali {
+namespace {
+
+template <typename Out, typename In>
+using TheKernel = kernels::PasteCpu<Out, In>;
+
+}  // namespace
+
+
+DALI_SCHEMA(MultiPaste)
+.DocStr(R"code(Performs multiple pastes from image batch to each of outputs
+
+This operator can also change the type of data.)code")
+.NumInput(1)
+.InputDox(0, "images", "3D TensorList", R"code(Batch of input images.
+
+Assumes HWC layout.)code")
+.AddArg("in_ids", R"code(1D TensorList of type int.
+
+Indexes from what inputs to paste data in each iteration.)code", DALI_INT32, true)
+.AddArg("in_anchors", R"code(2D TensorList of type int
+
+Absolute values of LU corner of the selection for each iteration.)code", DALI_INT32, true)
```

Comment:
Improvement suggestion: make it optional - if left out, use [0,0] as anchor.

Reply:
Done, also made optional for out_anchors